guides/request_lifecycle.md

```diff
@@ -117,7 +117,11 @@ defmodule HelloWeb.HelloHTML do
 end
 ```
 
-Now in order to add templates to this view, we can either define them as function components directly in the module:
```

Comment:
This "either ..." was missing its friend, so i *colocated* both parts, functions and files then expand on them.